build/babel.config.js

```diff
@@ -3,6 +3,7 @@
 module.exports = function(mode) {
   const plugins = [
     '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-proposal-class-properties',
```

Comment:
This is not strictly related to React, but this allows to use simplified ES class syntax, which is handy for state, static properties and autobound methods. See [this article](https://michalzalecki.com/react-components-and-class-properties/) for examples.